setup.cfg

```diff
@@ -39,7 +38,7 @@ include_package_data = True
 # [options.extras_require]
 
 [options.package_data]
-* = configs/*/*.yml
+* = configs/*/*.yml, optional_requirements.txt
```

Comment:
It seems like this can be removed. After commenting out the `options.package_data` section, I tested
```
python -m build --sdist --wheel --outdir dist/ .
```
and then installing both the `.whl` and `.tar.gz` files and directly
```
pip install .
```
from the main PeekingDuck directory. `optional_requirements.txt` and the `.yml` files are found in the installed package. Perhaps we can remove the `options.package_data` section.

Reply:
Okay let's remove it, and I guess the Boolean above can be removed too?

Reply:
Apparently `include_package_data = True` is still necessary, `optional_requirements.txt` and `.yml` files could not be found when I commented that out.